doc/classes/PathFollow2D.xml

```diff
@@ -18,9 +18,6 @@
 		<member name="h_offset" type="float" setter="set_h_offset" getter="get_h_offset" default="0.0">
 			The node's offset along the curve.
 		</member>
-		<member name="lookahead" type="float" setter="set_lookahead" getter="get_lookahead" default="4.0">
```

Comment:
Isn't this compat-breaking?

Reply:
Technically it does break compatibility: GDScript/C# code that refers to it will break, and I think this might impact the GDExtension ABI even for code that doesn't use it (but not sure about that, CC @godotengine/gdextension).

Since the property really did nothing, for a minor release it's acceptable I think, as long as we properly document it as a removal in the CHANGELOG.md (so I'll add the `breaks compat` label so we can keep track of it).

Reply:
Okay thanks, I marked breaks compat.